clients/Admin/package.json

```diff
@@ -44,10 +44,10 @@
     "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^12.0.4",
-    "@angular/cli": "^12.0.4",
-    "@angular/compiler-cli": "^12.0.4",
-    "@angular/language-service": "^12.0.4",
+    "@angular-devkit/build-angular": "12.2.16",
+    "@angular/cli": "12.2.16",
```

Comment:
Thank you. There seems to be a breaking change. Can you please make the exact changes in the `Landing/package.json` as well?

Reply:
The problem is not in Landing since it is not using coreui.  I have updated landing anyway to be consistent.